EIPS/eip-6228.md

```diff
@@ -0,0 +1,233 @@
+---
+eip: 6228
+title: Extreme ЕRС-20
+description: meta-transaction token (MTT)
```

Comment:
These are both way too vague. You need to give readers some kind of idea about what your proposal is about.